VirtoCommerce.Platform.Data.Azure/AzureBlobProvider.cs

```diff
@@ -375,6 +356,24 @@ private CloudBlobContainer GetBlobContainer(string name)
             return retVal;
         }
 
+        private BlobInfo ConvertCloudBlobToBlobInfo(ICloudBlob cloudBlob)
+        {
+            var relativeUrl = cloudBlob.Uri.LocalPath;
+            var absoluteUrl = GetAbsoluteUrl(cloudBlob.Uri.PathAndQuery);
+            var fileName = Path.GetFileName(Uri.UnescapeDataString(cloudBlob.Uri.ToString()));
```

Comment:
I guess my variant more readable for a technically unready person in general:
var cloudBlobUri = cloudBlob.Uri.ToString();
var unescapeDataString = Uri.UnescapeDataString(cloudBlobUri);
var fileName = Path.GetFileName(unescapeDataString);

This should be readable as a book )

Reply:
This is unnecessary.